modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -7,29 +7,24 @@
 [id="importing-untrusted-tls-certificates_{context}"]
 = Importing untrusted TLS certificates to {prod-short}
 
-External communications between {prod-short} components are, by default, encrypted with TLS. Communications of {prod-short} components with external services such as proxies, source code repositories, and identity provider may require using TLS. Those communications require the use of TLS certificates signed by trusted Certificate Authorities.
+By default, external communications between {prod-short} components are encrypted with TLS. Communications of {prod-short} components with external services such as proxies, source code repositories, and identity provider might also require TLS. All communications encrypted with TLS require the use of TLS certificates signed by trusted Certificate Authorities (CA).
 
-NOTE: When the certificates used by {prod-short} components or by an external service are signed by an untrusted CA, it can be necessary to import the CA certificate in the {prod-short} installation so that every {prod-short} component will consider them as signed by a trusted CA.
+When the certificates used by {prod-short} components or by an external service are signed by an untrusted CA, you must import the CA certificate into the {prod-short} instance so that every {prod-short} component treats the certificates as signed by a trusted CA. You have to do this in the following cases:
 
-Typical cases that may require this addition are:
-
-* When the underlying {orch-name} cluster that uses TLS certificates signed by a CA that is not trusted.
-* When {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
+* The underlying {orch-name} cluster uses TLS certificates signed by an untrusted CA.
+* {prod-short} server or workspace components connect to external services such as {identity-provider} or a Git server that use TLS certificates signed by an untrusted CA.
```

Comment:
Today Keycloak is not the default OIDC provider anymore. Hence we may need to change that in https://github.com/eclipse-che/che-docs/blob/main/antora.yml#L29.

Reply:
Is there something I can do? Who's the current default provider?

Reply:
@l0rd, what should be the new value for these attributes/variables?
```
    identity-provider-id: keycloak
    identity-provider: Keycloak
```
@deerskindoll, btw, looks like we might need to update the downstream docs as well; currently, this is downstream:
```
:identity-provider-id: keycloak
:identity-provider: RH-SSO
```

Reply:
I propose these: 
```
    identity-provider-id: openshift-oauth
    identity-provider: OpenShift OAuth
```

Reply:
Feedback from Anatolii:
![image](https://user-images.githubusercontent.com/78564304/182167383-daa0734b-65e6-49eb-8847-6ec0c2dbff15.png)

Reply:
I'm opening a RHDEVDOCS Jira issue for this.